src/pages/blog/starlingx-debian-technology-preview-in-release-7.md

```diff
@@ -0,0 +1,376 @@
+---
+templateKey: blog-post
+title: Debian Technology Preview in Release 7
+author: Frank Miller and Eric Macdonald
+date: 2022-07-28T16:23:52.741Z
+category:
+  - label: News & Announcements
+    id: category-A7fnZYrE1
+---
+
+With support for the CentOS Distribution being discontinued, StarlingX will move
+to the Debian OS Distribution. Here's how to get ready.<!– more –>
+
+Debian is a well-established Linux Distribution supported by a large and mature
+open-source community and used by hundreds of commercial organizations,
+including Google. When fully transitioned to Debian, StarlingX will have full
+functional equivalence to the current CentOS-based versions of StarlingX.
+
+# Rollout
```

Comment:
This section reads like a product roadmap and less like a blog post. The post is also a little long, therefore I would suggest to replace it with a reference to this information. Is this available in a spec, StoryBoard story or on a wiki page?

I think it would be a good approach to describe it here that moving to the Debian-based platform will span across multiple release cycles, due to the size of the task. The first, experimental version is now available in 7.0 with limited scope, which you can take blog post to your description below.